### DISCUSSION TITLE
✅ SSH tunnel stalled if app is in the background

### DISCUSSION THREAD

**Discussion by txdoesker** [isontheline/pro.webssh.net#800] [Q&A]
Hi,

I use  a ssh tunnel, to connect to my fhem server.
I connect to it with safari on https://localhost:4711.
Safari seems to get stalled during load of the  web page.
But if I click webSSH in the foreground an switch back to safari, the page is loaded.

ssh tunnels, not getting cpu time, while webSSH is in the background, makes no sense to me.
Is there any setup option, I missed?

**Comment by isontheline** [comment]
Hi @txdoesker 👋

Yes you need to enable "VPN-Over-SSH" inside your connection settings.

Have a nice day ☀️

**Reply by txdoesker** [reply]
Hello @isontheline, 

what a perfect integration. Works like a charm.
Just upgraded to the PRO-Version.

Thank you  very much

**Reply by isontheline** [reply]
You're Welcome!

Thank you so much for your support and your purchase 🙏
